src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+        Ball chosenOne = new Ball();
+        chosenOne.setColor(getRandomColor());
+        Random random = new Random();
+        int randomNumber = random.nextInt(101);

```

Comment:
Magic number 101 should be replaced with a named constant to clarify its purpose and improve code readability.